src/plone/restapi/bbb.py

```diff
@@ -0,0 +1,39 @@
+try:
+    from plone.base.defaultpage import is_default_page  # noqa: F401
+    from plone.base.interfaces import IConstrainTypes  # noqa: F401
+    from plone.base.interfaces import IEditingSchema  # noqa: F401
+    from plone.base.interfaces import IImagingSchema  # noqa: F401
+    from plone.base.interfaces import ILanguage  # noqa: F401
+    from plone.base.interfaces import IMailSchema  # noqa: F401
+    from plone.base.interfaces import INavigationSchema  # noqa: F401
+    from plone.base.interfaces import INonInstallable  # noqa: F401
+    from plone.base.interfaces import INonStructuralFolder  # noqa: F401
+    from plone.base.interfaces import IPloneSiteRoot  # noqa: F401
+    from plone.base.interfaces import ISearchSchema  # noqa: F401
+    from plone.base.interfaces import ISecuritySchema  # noqa: F401
+    from plone.base.interfaces import ISelectableConstrainTypes  # noqa: F401
+    from plone.base.interfaces import ISiteSchema  # noqa: F401
+    from plone.base.utils import base_hasattr  # noqa: F401
+    from plone.base.utils import safe_callable  # noqa: F401
+    from plone.base.utils import safe_hasattr  # noqa: F401
+    from plone.base.utils import safe_text  # noqa: F401
+except ImportError:
+    # BBB Plone 5.2
+    from Products.CMFPlone.defaultpage import is_default_page  # noqa: F401
+    from Products.CMFPlone.interfaces import IConstrainTypes  # noqa: F401
+    from Products.CMFPlone.interfaces import IEditingSchema  # noqa: F401
+    from Products.CMFPlone.interfaces import IImagingSchema  # noqa: F401
+    from Products.CMFPlone.interfaces import ILanguage  # noqa: F401
+    from Products.CMFPlone.interfaces import IMailSchema  # noqa: F401
+    from Products.CMFPlone.interfaces import INavigationSchema  # noqa: F401
+    from Products.CMFPlone.interfaces import INonInstallable  # noqa: F401
+    from Products.CMFPlone.interfaces import INonStructuralFolder  # noqa: F401
+    from Products.CMFPlone.interfaces import IPloneSiteRoot  # noqa: F401
+    from Products.CMFPlone.interfaces import ISearchSchema  # noqa: F401
+    from Products.CMFPlone.interfaces import ISecuritySchema  # noqa: F401
+    from Products.CMFPlone.interfaces import ISelectableConstrainTypes  # noqa: F401
+    from Products.CMFPlone.interfaces import ISiteSchema  # noqa: F401
+    from Products.CMFPlone.utils import base_hasattr  # noqa: F401
+    from Products.CMFPlone.utils import safe_callable  # noqa: F401
+    from Products.CMFPlone.utils import safe_hasattr  # noqa: F401
+    from Products.CMFPlone.utils import safe_text  # noqa: F401
```

Comment:
Rather than filling the whole file with `# noqa: F401` you can add on setup.cfg:

```ini
[flake8]
per-file-ignores =
    src/plone/restapi/bbb.py:F401
```

Reply:
I pushed such a change and remove the comments, it should be all good 🤞🏾 🍀